src/CalibrateCMP/OptimizationUtils.jl

```diff
@@ -47,11 +47,17 @@ end
 
 # EKP
 function calibrate(::EKPStyle, priors, config, ref_stats_list::Vector{ReferenceStatistics}; verbose::Int = 3)
-    @assert config["process"]["EKP_method"] == "EKI" || config["process"]["EKP_method"] == "UKI"
+    @assert config["process"]["EKP_method"] == "EKI" ||
+            config["process"]["EKP_method"] == "UKI" ||
+            config["process"]["EKP_method"] == "ETKI"
 
     u_names = collect(keys(config["prior"]["parameters"]))
     n_iter = config["process"]["n_iter"]
-    n_ensemble = config["process"]["EKP_method"] == "EKI" ? config["process"]["n_ens"] : 2 * length(u_names) + 1
+    if config["process"]["EKP_method"] == "EKI" || config["process"]["EKP_method"] == "ETKI"
+        n_ensemble = config["process"]["n_ens"]
+    else
+        n_ensemble = 2 * length(u_names) + 1
+    end
```

Comment:
Here I would write the if statement for UKI and the else statement for others. The reason is that it should be clear that 2p+1 is for UKI and other methods use the number set bu the user.